ansible_collections/arista/avd/molecule/eos_cli_config_gen_deprecated_vars/documentation/devices/host1.md

```diff
@@ -1829,6 +1829,10 @@ qos profile test
 | ----------------------- |
 | ethernet1 |
 
+| Bindings Timeout (s) | SSL Profile | SSL Connection Lifetime |
```

Comment:
Looking at this I feel like we should do the following instead (one table)
where we join the local interfaces using `<br>`

```
| Local interfaces | Binding Timeout(s) | SSL Profile | SSL Connection Lifetime |
```

What do you think @harshitk-arista ?

Reply:
This one is deprecated so I'll skip this. Will make this change for the non-deprecated bits.